src/script/components/participantAvatar/ServiceAvatar.tsx

```diff
@@ -61,16 +60,16 @@ const ServiceAvatar: React.FunctionComponent<ServiceAvatarProps> = ({assetReposi
         }}
         data-uie-name="element-avatar-service-icon"
       >
-        <svg
+        <NamedIcon
+          name="service-icon"
           viewBox="0 0 32 32"
```

Comment:
Can this be width & height 32?

Reply:
nope, only viewBox is needed and not width attribute on element